### ISSUE TITLE
To help improve proper tagging of images, show pop-up help on mouseover of tag

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#2585]
As I've been diving more deeply into our data via Sidewalk Gallery explorations, I've noticed some inconsistent tagging (which I hope we'll make editable, see https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2575).

But I also think we can improve tagging, in general, by providing better _in situ_ information to the user when they are labeling. So, for example, when labeling a curb ramp, as the user glides over the various tags, we could show parts of the image below appropriately highlighted (e.g., when user mouses over "not enough landing space", we would highlight that part of the curb ramp diagram). We would want to simplify this diagram, of course, by removing things that are not relevant to Project Sidewalk (because they are too hard to reliably assess with our methods).

![image](https://user-images.githubusercontent.com/1621749/122449817-25517100-cf5b-11eb-97d2-30ce46fd8319.png)

I think @isavin12 could be a great fit for this.

**Comment by misaugstad** [comment]
Yeah that's a good look. @uditpatwal recently added tooltip examples (similar to what we have for severity), which this sounds similar to! But showing it on a diagram might be more helpful than just showing an example. Although I guess that for landing space, @uditpatwal added a bit of annotation to make it more useful:
![23](https://user-images.githubusercontent.com/6518824/122458257-77e35b00-cf64-11eb-9789-5a365e03d54d.png)

**Comment by misaugstad** [comment]
@jonfroehlich so @uditpatwal did recently add something similar to this when he added in tooltips on the tags on the audit page. Is this ticket about improving them by replacing some/all of the example pictures with diagrams that might be more informative? Or did you just not realize that we've recently added tooltips when making the ticket? Should the ticket be about adding those tooltips into Gallery and/or Validation once we make data editable?

I'm just not exactly sure what the action item is for this issue.

**Comment by jonfroehlich** [comment]
Yes, should be added to Gallery and Validation imo.

I still think there is room for possibly doing diagrammatic rather than pictorial versions of the pop-ups too (as my post originally suggests). I have not seen @uditpatwal's feature, so maybe his work could indeed just be applied to all areas where we have severity info.

Is there a place to easily learn more about and see  @uditpatwal's popups (e.g., a single screenshot that shows all the various popups, etc.)

**Comment by misaugstad** [comment]
There isn't a single screenshot or anything, no. But if you just open the audit page, you can place one label of each label type and hover over the tags to quickly get an idea of where we're at.

**Comment by jonfroehlich** [comment]
Wow, this is really neat @uditpatwal. I just reviewed them. Great job!! Only thing I could have asked for us a one-stop place to view and review all of these pop-ups. To address this, here they are with some comments :) (Update: I couldn't finish this because the server just went unresponsive... perhaps due to an upgrade that Jason's doing... so this list is incomplete).

## Curb Ramp Pop-ups
![image](https://user-images.githubusercontent.com/1621749/123703299-0c797300-d819-11eb-9627-ca42471039f4.png)

![image](https://user-images.githubusercontent.com/1621749/123703401-2d41c880-d819-11eb-96ca-e2722a23083e.png)

![image](https://user-images.githubusercontent.com/1621749/123703431-36cb3080-d819-11eb-9859-f90537f04bd2.png)

![image](https://user-images.githubusercontent.com/1621749/123704101-26678580-d81a-11eb-94c9-6e03d3eeeb82.png)
![image](https://user-images.githubusercontent.com/1621749/123704194-48f99e80-d81a-11eb-89c4-edd900f19733.png)

![image](https://user-images.githubusercontent.com/1621749/123704213-4f881600-d81a-11eb-9594-842018de346d.png)

## Missing Curb Ramps

Love the use of the arrow here. I think we could add an arrow to the images above (for Curb Ramps) to better bring the user's attention to the specific problem.

![image](https://user-images.githubusercontent.com/1621749/123704351-69c1f400-d81a-11eb-96a9-65eae4f09488.png)

![image](https://user-images.githubusercontent.com/1621749/123704527-91b15780-d81a-11eb-9dd7-5a40a8145433.png)

![image](https://user-images.githubusercontent.com/1621749/123704569-a130a080-d81a-11eb-93ce-6bd9a2768135.png)

## Pop-up mages that could be improved

This one doesn't really represent "points into traffic" for me. Typically, these types of ramps are the ones that diagonally position the pedestrian into traffic.

![image](https://user-images.githubusercontent.com/1621749/123703397-2b780500-d819-11eb-9125-451f840309f5.png)

I don't agree with this one. Not enough context to determine if the ramp is needed. What do you think @misaugstad? 

![image](https://user-images.githubusercontent.com/1621749/123704590-a7268180-d81a-11eb-8552-6db81a375034.png)


## Other image sources
In our recently submitted Gallery poster, I essentially must have done the exact same exploration with the Gallery tool as @uditpatwal. My analysis is maybe not as complete but some of my pictures feel more exemplary and easy-to-understand.

![image](https://user-images.githubusercontent.com/1621749/123703733-a6d9b680-d819-11eb-8d5c-4e85471ad0e0.png)

I have many other examples I've been collecting in a PowerPoint deck, which I was building for our ASSETS'21 poster submission on Sidewalk Gallery.

And, of course, we have these Google Slide decks too, which @gari01234 helped us build (and, I think, he incorporated into some of the teaching curriculum in Mexico):

- [Curb Ramps: Correct and Incorrect Examples](https://docs.google.com/presentation/d/1HDIw0Nzy72s0HdIn4AMsZcIgE3C3YjemrJeCZkFDHxQ/edit#slide=id.g9a59c21d14_0_72)
- [Missing Curb Ramps: Correct and Incorrect Examples](https://docs.google.com/presentation/d/1_INkXmBoafq7rHtUmDaR5n9TSrBA62g69WWfOVZWYWA/edit#slide=id.g9a639fa2a5_0_210)
- [Surface Problems: Correct and Incorrect Examples](https://docs.google.com/presentation/d/1QfTEOF0sJXyaB_yvfy9oI1POOQrgysMaHcYaZsKgf7s/edit#slide=id.p)
- [Obstacles: Correct and Incorrect Examples](https://docs.google.com/presentation/d/1E8f_dqIIhjPCGI9z69FCxguwgtRcH92Y77-mV1xduOs/edit#slide=id.g9a569457c5_0_6)
- [No Sidewalk: Correct and Incorrect Examples](https://docs.google.com/presentation/d/15TOCN0L09Yxcn8taUs92fMIqNxB9LDXvhB6A_56uKys/edit#slide=id.p)

**Comment by misaugstad** [comment]
closing via #3609
